src/NzbDrone.Core.Test/Download/DownloadClientTests/Blackhole/TorrentBlackholeFixture.cs

```diff
@@ -185,7 +185,7 @@ public void Download_should_replace_illegal_characters_in_title()
             var illegalTitle = "Saturday Night Live - S38E08 - Jeremy Renner/Maroon 5 [SDTV]";
```

Comment:
Do these need to be changed? Magnet URI may need to be changed as well if it links to this torrent.